packages/cli/src/snapwp.cjs

```diff
@@ -64,8 +69,13 @@ const DEFAULT_PROJECT_PATH = './snapwp-app';
 		// Step 6: Update package versions
 		await updatePackageVersions( projectDirPath );
 
-		// Step 7: Print instructions
-		printSuccessMessage( projectDirPath, false );
+		// Step 7: Install dependencies (skip if flag is set)
+		const installSuccess = await runNpmInstall( projectDirPath, {
```

Comment:
Rather than getting an installSuccess as an bool, Let `runNpmInstall` throw. Derive if args for `printSuccessMessage` from that.